### DISCUSSION TITLE
Management_Agent

### DISCUSSION THREAD

**Discussion by Jenks-ho** [oracle-quickstart/oci-kubernetes-monitoring#61] [Q&A]
The management agent is up and running on the cluster but has not registered in the console. Also, there are no metrics being posted - I guess these are related but any ideas/help would be much appreciated.

**Comment by palukuri** [comment]
@niravg75 can you help with this

**Comment by niravg75** [comment]
You need to check if the Pods (vcn) can reach OCI services.

**Reply by Jenks-ho** [reply]
The cluster is able to send data to Logging Analytics and we are able to get dashboards populated so I guess that part is good. We have non OKE compute instances in the same compartment and VCN using the Management Agent for Stack Monitoring and these do show up in the console. I will re-check everything again.
